src/index.tsx

```diff
@@ -18,7 +18,7 @@ const onBeforeLift = () => {
 };
 // Some amount of meta data is delivered to the app by the server
 /* tslint:disable:no-string-literal */
-const migMeta = window['_mig_meta'];
+const migMeta = JSON.parse(atob(window['_mig_meta']));
```

Comment:
This probably should have been a b64 value from the beginning. Solves a lot of problems.